Resources/Prototypes/Entities/Mobs/Species/arachne.yml

```
@@ -22,6 +22,7 @@
       color: "#e8b59b"
       sprite: Mobs/Species/Human/parts.rsi
       state: head_m
+    - map: [ "enum.HumanoidVisualLayers.Snout" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)